docs/reference-guides/backup-restore-configuration/backup-configuration.md

```diff
@@ -79,9 +85,11 @@ The S3 storage location contains the following configuration fields:
 1. **Skip TLS Verifications** (optional): Set to true if you are not using TLS.
 
 
+#### YAML Directive Fields
+
 | YAML Directive Name | Description | Required |
 | ---------------- | ---------------- | ------------ |
-| `credentialSecretName` |  If you need to use the AWS Access keys Secret keys to access s3 bucket, create a secret with your credentials with keys and the directives `accessKey` and `secretKey`. It can be in any namespace as long as you provide that namespace in `credentialSecretNamespace`. An example secret is [here.](#example-credentialsecret) This directive is unnecessary if the nodes running your operator are in EC2 and set up with IAM permissions that allow them to access S3, as described in [this section.](#iam-permissions-for-ec2-nodes-to-access-s3)  |  |
+| `credentialSecretName` |  If you need an AWS access key or secret key to access an S3 bucket, [create a secret](../../how-to-guides/new-user-guides/kubernetes-resources-setup/secrets.md) using your credentials, with keys and directives named `accessKey` and `secretKey`. The secret can be in any namespace. An example secret is [here](#example-credentialsecret). This directive is unnecessary if the nodes running your operator are in EC2 and assigned [IAM permissions to access S3](#iam-permissions-for-ec2-nodes-to-access-s3). |  |
```

Comment:
This section is for the YAML fields, but we don't mention YAML at all on secrets.md so linking to it might be confusing.